app/services/moves/import_people.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module Moves
+  class ImportPeople
+    attr_accessor :items
+
+    def initialize(items)
+      self.items = items
+    end
+
+    def call
+      import_people
+    end
+
+  private
+
+    def import_people
+      people_nomis_prison_numbers = items.map { |item| item.fetch(:person_nomis_prison_number) }
+      people = NomisClient::People.get(people_nomis_prison_numbers).map { |p| [p.fetch(:prison_number), p] }.to_h
+      alerts = NomisClient::Alerts.get(people_nomis_prison_numbers).group_by { |p| p.fetch(:offender_no) }
+      personal_care_needs = NomisClient::PersonalCareNeeds
+                            .get(people_nomis_prison_numbers)
+                            .group_by { |p| p.fetch(:offender_no) }
```

Comment:
Most of these look like they can be extracted in other private methods you can call as you do with the respective variables. This would make the method much less dense and defer fetching data to well named methods, IMHO adding clarity.

Reply:
I agree 💯 , this code is not nice and not readable.
I've just copy/paste it from a `Moves::Importer`,
no time for refactoring atm.

Reply:
Let's not copy and paste - it leads to lots of problems. 
The existing code has an import_people method that could probably just be made public. 
If we don't have time do it it right now, we will never have time later when we're really under pressure.